agent/util-scripts/gold/pbench-metadata-log/test-03.txt

```diff
@@ -1,9 +1,18 @@
 +++ Running test-03 pbench-metadata-log --dir=/var/tmp/pbench-test-utils/pbench/tmp beg
+/bin/ls: cannot access '/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/tools-default': No such file or directory
```

Comment:
Where is this output coming from and is it necessary?

Reply:
Same bug: the mocked scp was creating the tools files in the wrong directory.